src/goto-harness/recursive_initialization.h

```diff
@@ -22,6 +22,7 @@ Author: Diffblue Ltd.
 #include "function_harness_generator_options.h"
 #include "goto_harness_generator.h"
 
+#define GOTO_HARNESS_PREFIX "GOTO_HARNESS"
```

Comment:
:grey_question: Why not 
```suggestion
#define GOTO_HARNESS_PREFIX CPROVER_PREFIX "_GOTO_HARNESS"
```

Reply:
@thk123 anything starting with `CPROVER_PREFIX` is filtered out and ignored by `dump_c`, that's the only reason we added a goto harness specific prefix.

Reply:
I would have thought that as this is an "implementation" prefix, it should be `__GOTO_HARNESS` ? (i.e. double underscore at the front) ?

Reply:
We could certainly add a `__` in front.